sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample9_SingleLabelClassify.md

```diff
@@ -0,0 +1,93 @@
+
+To perform Custom Single Label Classification in one or multiple documents, set up a `SingleLabelClassifyAction` and call `StartAnalyzeActionsAsync` on the documents. The result is a Long Running Operation of type `AnalyzeActionsOperation` which polls for the results from the API.
+
+```C# Snippet:TextAnalyticsSingleLabelClassifyAsync
+// Get input document.
+string document = @"I need a reservation for an indoor restaurant in China. Please don't stop the music. Play music and add it to my playlist.";
+
+// Prepare analyze operation input. You can add multiple documents to this list and perform the same
+// operation to all of them.
+var batchInput = new List<string>
+{
+    document
+};
+
+// Set project and deployment names of the target model
+// To train a model to classify your documents, see https://aka.ms/azsdk/textanalytics/customfunctionalities
+string projectName = "<projectName>";
+string deploymentName = "<deploymentName>";
+
+var singleLabelClassifyAction = new SingleLabelClassifyAction(projectName, deploymentName);
+
+TextAnalyticsActions actions = new TextAnalyticsActions()
+{
+    SingleLabelClassifyActions = new List<SingleLabelClassifyAction>() { singleLabelClassifyAction }
+};
+
+// Start analysis process.
+AnalyzeActionsOperation operation = await client.StartAnalyzeActionsAsync(batchInput, actions);
+
+await operation.WaitForCompletionAsync();
+```
+
+The returned `AnalyzeActionsOperation` contains general information about the status of the operation. It can be requested while the operation is running or when it has completed. For example:
+
+```C# Snippet:TextAnalyticsSingleLabelClassifyOperationStatus
+// View operation status.
+Console.WriteLine($"AnalyzeActions operation has completed");
+Console.WriteLine();
+
+Console.WriteLine($"Created On   : {operation.CreatedOn}");
+Console.WriteLine($"Expires On   : {operation.ExpiresOn}");
+Console.WriteLine($"Id           : {operation.Id}");
+Console.WriteLine($"Status       : {operation.Status}");
+Console.WriteLine($"Last Modified: {operation.LastModified}");
+Console.WriteLine();
+```
+
+To view the final results of the long-running operation:
+
+```C# Snippet:TextAnalyticsSingleLabelClassifyAsyncViewResults
+// View operation results.
+await foreach (AnalyzeActionsResult documentsInPage in operation.Value)
+{
+    IReadOnlyCollection<SingleLabelClassifyActionResult> singleClassificationActionResults = documentsInPage.SingleLabelClassifyResults;
+
+    foreach (SingleLabelClassifyActionResult classificationActionResults in singleClassificationActionResults)
+    {
+        Console.WriteLine($" Action name: {classificationActionResults.ActionName}");
+        foreach (SingleLabelClassifyResult documentResults in classificationActionResults.DocumentsResults)
+        {
+            Console.WriteLine($"  Class label \"{documentResults.Classification.Category}\" predicted with a confidence score of {documentResults.Classification.ConfidenceScore}.");
```

Comment:
Should documentResults be a list here like in the multi label sample? I thought I read in the changelog that you merged the result types, but correct me if I'm wrong.

Reply:
It will be once I merged. I guess the CHANGELOG is in between right now, but I'll make sure to fix this so that it would be a collection, yes. I opened https://github.com/Azure/azure-sdk-for-net/issues/30089 to track.